cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,162 @@
+/// <reference types="Cypress" />
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+And('user clicks on Image content type', () => {
+    cy.get(`ul.admin-list span.label:contains('Image')`).then($el => {
+        const content= $el[1];
+        cy.get(content).parent().click({ force: true });
+    })
+});
+
+And('user types {string} into Caption text field', (value) => {
+    cy.getIframeBody('iframe.cke_wysiwyg_frame.cke_reset').find('p').type(value);
+})
+
+And('user uploads test image {string}', (fileName) => {
+    cy.fixture(fileName, { encoding: null }).as('fixture')
+    if (fileName === "main_image.jpg")
+    {cy.get('input#edit-field-media-image-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "feature_card_image.jpg")
+    {cy.get('input#edit-field-override-img-featured-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "thumbnail_image.jpg")
+    {cy.get('input#edit-field-override-img-thumbnail-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "social_media_image.jpg")
+    {cy.get('input#edit-field-override-img-social-media-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "panoramic_image.jpg")
+    {cy.get('input#edit-field-override-img-panoramic-0-upload').selectFile('@fixture');}
+
+    cy.wait(2000);
```

Comment:
remove wait from the step - if you need to wait use 
And browser waits

in the feature file

Reply:
done